<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameBoard</class>
 <widget class="QWidget" name="GameBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>603</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>251</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="restartButton">
      <property name="text">
       <string>重新开始</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="repentanceButton">
      <property name="text">
       <string>悔棋</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="giveUpButton">
      <property name="text">
       <string>投降</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="restartNewGameButton">
      <property name="text">
       <string>开始新游戏</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
